src/app/newSettings/Sections/Workspace/Members/components/ActivityTab.tsx

```diff
@@ -0,0 +1,7 @@
+import { ActivityTabProps } from '../../../../types';
+
+const ActivityTab = ({ user, activity }: ActivityTabProps): JSX.Element => {
+  return <div>ACTIVITY TAB</div>;
```

Comment:
Do the names of the tabs need translations or do they always have a fixed name in English?

Reply:
This is only temporary just to display something, here I have to create the whole Activity view.